gradle.properties

```diff
@@ -1,2 +1,5 @@
 #DCSTOOL_HOME
```

Comment:
this should be a `gradle.properties.example` with `gradle.properties` excluded from the build.
Less risk of credential spillage that way.